isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -160,15 +152,8 @@ namespace Isis {
       points->clear();
     }
 
-    if (cubeGraphNodes) {
-      QHashIterator< QString, ControlCubeGraphNode * > i(*cubeGraphNodes);
-      while (i.hasNext()) {
-        i.next();
-        delete(*cubeGraphNodes)[i.key()];
-        (*cubeGraphNodes)[i.key()] = NULL;
-      }
-      cubeGraphNodes->clear();
-    }
+    m_controlGraph.clear();
+    m_vertexMap.clear();
```

Comment:
out of an abundance of caution, we should swap these. There are null pointers in the map for a brief moment with this order.